packages/server/src/context.ts

```diff
@@ -59,8 +59,12 @@ export class AuthenticatedRequestContext extends RequestContext {
     return this.authState.login;
   }
 
-  get profile(): Reference<ProfileResource> {
-    return this.membership.profile as Reference<ProfileResource>;
+  get profile(): Reference<ProfileResource | Bot | ClientApplication> {
+    return this.membership.profile;
+  }
+
+  get realProfile(): Reference<ProfileResource | Bot | ClientApplication> {
```

Comment:
I don't love this name, especially without comments or documentation.

How would you feel about removing this getter, and just using `.authState.membership.profile` at the call site?

The convention being:  if you know what you're doing, and you know what you need, then you should use `.authState` directly.